e2e/pages/generalApplication/checkYourAnswers.page.js

```diff
@@ -5,24 +5,27 @@ module.exports = {
   fields: {
     checkAnswerForm: {
       classname: '.check-your-answers',
-      changeLink: '.case-field-change a',
-      uploadEvidenceLink: '.collection-field-table a'
```

Comment:
why did you reverted back my changes @pliao-hmcts

Reply:
because it doesn't work on my local